language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -508,3 +509,24 @@ test('stream close behaviour', async () => {
     'close',
   ]);
 });
+
+test('party API', async () => {
+  const p = (id: string): PartyInfo => ({identifier: id, displayName: id, isLocal: true});
+  const ledger = new Ledger({token: ALICE_TOKEN, httpBaseUrl: httpBaseUrl()});
+  const parties = await ledger.fetchPartiesInfo([ALICE_PARTY, "unknown"]);
+  expect(parties).toEqual([p("Alice"), null]);
+  const rev = await ledger.fetchPartiesInfo(["unknown", ALICE_PARTY]);
+  expect(rev).toEqual([null, p("Alice")]);
+
+  const allParties = await ledger.fetchAllPartiesInfo();
+  expect(_.sortBy(allParties, [(p: PartyInfo) => p.identifier])).toEqual([p("Alice"), p("Bob")]);
+
+  const newParty1 = await ledger.allocateParty({});
+  const newParty2 = await ledger.allocateParty({displayName: "Carol"});
+  await ledger.allocateParty({displayName: "Dave", identifierHint: "Dave"});
+
+  const allPartiesAfter = _.map(await ledger.fetchAllPartiesInfo(), (pi) => pi.identifier);
```

Comment:
```suggestion
  const allPartiesAfter = (await ledger.fetchAllPartiesInfo()).map((pi) => pi.identifier);
```
Shouldn’t that be equivalent?

Reply:
Yes it is. I guess I just got carried away while doing the sort thing. `.map` does create a new array so no good reason to use `lodash` here.